schema.graphql

```diff
@@ -704,3 +704,25 @@ type DaiApproval @entity {
   " Timestamp of the update approval "
   updatedAt: BigInt!
 }
+
+type SaleAuction @entity {
```

Comment:
does it make sense to add an `active` field? I saw that the contract stores that information